src/conf_mode/protocols_isis.py

```diff
@@ -203,6 +203,37 @@ def verify(isis):
         if list(set(global_range) & set(local_range)):
             raise ConfigError(f'Segment-Routing Global Block ({g_low_label_value}/{g_high_label_value}) '\
                               f'conflicts with Local Block ({l_low_label_value}/{l_high_label_value})!')
+        
+    # Check for a blank or invalid value per prefix
+    if dict_search('segment_routing.prefix', isis):
+        for prefix in isis['segment_routing']['prefix']:
+            if 'absolute' in isis['segment_routing']['prefix'][prefix]:
+                try:
+                    segment_label_value = int(isis['segment_routing']['prefix'][prefix]['absolute']['value'])
+                    if not 16 <= segment_label_value <= 1048576:
+                        raise ConfigError(f'Segment routing prefix {prefix} absolute value {segment_label_value} is invalid.')
+                except KeyError:
+                    raise ConfigError(f'Segment routing prefix {prefix} absolute value cannot be blank.')
+            elif 'index' in isis['segment_routing']['prefix'][prefix]:
+                try:
+                    segment_label_value = int(isis['segment_routing']['prefix'][prefix]['index']['value'])
+                    if not 0 <= segment_label_value <= 65535:
```

Comment:
This is already handled by the XML definition.

```xml
        <node name="index">
          <properties>
            <help>Specify the index value of prefix segment/label ID</help>
          </properties>
          <children>
            <leafNode name="value">
              <properties>
                <help>Specify the index value of prefix segment/label ID</help>
                <valueHelp>
                  <format>u32:0-65535</format>
                    <description>The index segment/label ID value</description>
                </valueHelp>
                <constraint>
                  <validator name="numeric" argument="--range 0-65535"/>
                </constraint>
              </properties>
            </leafNode>
          ...
```

Reply:
As above...

Should I comment it out, or just remove completely?

Reply:
Please remove completely - no need to keep unused code that does not belong there.

Reply:
Will do sir. Thank you.